DependencyInjection/CmfRoutingExtension.php

```diff
@@ -371,14 +384,32 @@ private function loadConditionalEnhancer(array $config, ContainerBuilder $contai
         $container->addDefinitions($definitions);
     }
 
-    private function buildConditionalEnhancerDefintion($className, $source, $target, $map, $methods, $priority)
+    private function buildConditionalEnhancerDefintion($className, $source, $target, $maps, $priority)
     {
+        var_dump($maps);throw new \Exception;
```

Comment:
the PR does say WIP at least :-)

i needed to figure out what was going on to understand i have a total misconception.